src/App.js

```diff
@@ -18,14 +18,15 @@ import {
 library.add(faDotCircle, faMinusCircle, faPen, faSave, faTrash);
 
 class App extends Component {
+  componentDidMount() {}
   render() {
     return (
       <DragDropContextProvider backend={HTML5Backend}>
         <div className="container">
-          <h1>Test Structural Metadata Editor</h1>
-          <WaveformContainer />
+          <br />
```

Comment:
This too, let's get rid of.  If we need padding or margin around the component, let's let the consuming application handle that styling decision.

Reply:
@adamjarling Should I replace `<br />` with `<h1>Structural Metadata Editor</h1>`? So that the stand-alone application has a header. We can remove this in the packaging repo when importing to Avalon.

Reply:
Let's leave the header out.  We should start thinking of this now as an importable component, and the header is good to remove I'd say.